src/shared/components/base/gokb-textarea-field/gokb-textarea-field.vue

```diff
@@ -94,6 +99,20 @@
         }
       },
     },
+    watch: {
+      value () {
+        this.setDefRows()
+      }
+    },
+    mounted() {
+      this.setDefRows()
+    },
+    methods: {
+      setDefRows() {
+        let newRows = this.localValue?.match(/\n/g)?.length || 0
```

Comment:
This only stretches the textarea if newlines exist. We need are vertical stretch for text without newlines also.
We could operate with letter count and estimations but this would be quite ugly.